theme/static/js/generic-resource.js

```diff
@@ -460,16 +459,12 @@ $(document).ready(function () {
     // set if multiple files can be uploaded
     if (ALLLOW_MULTIPLE_FILE_UPLOAD === "True") {
         $("#file-multiple").text("Multiple file upload is allowed.");
-        $("#file-multiple-irods").text("Multiple file upload is allowed.");
     }
     else {
         $("#file-multiple").text("Only one file can be uploaded.");
-        $("#file-multiple-irods").text("Only one file can be uploaded.");
 
         if (FILE_COUNT > 0) {
-            $("#log-into-irods").hide();
             $("#sign-in-info").hide();
```

Comment:
might be able to remove `sign-in-info` as well?
I'm not sure what it was used for, but it looks like it might not exist anymore?